### ISSUE TITLE
Minishift openshift config view fails when node is targeted

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1227]
I have a problem seeing OpenShift configuration for node with command`minishift openshift config view --target node`. I am using Minishift v1.3.1+7c97e62 from master, the file might have been relocated in oc - tried with GA and there it is fine.

```
[agajdosi@dhcp-24-156 bin]$ ./minishift openshift config view --target node
Cannot display the OpenShift configuration: ssh command error:
command : docker exec -t origin cat /var/lib/origin/openshift.local.config/node-192.168.42.5/node-config.yaml
err     : exit status 1
output  : cat: /var/lib/origin/openshift.local.config/node-192.168.42.5/node-config.yaml: No such file or directory
```

**Comment by hferentschik** [comment]
Ahh, this is due to a change in the node name between OpenShift 1.5 and 3.6. I believe the node config is now always node-127.0.0.1.

**Comment by hferentschik** [comment]
Merged via pull request #1284

**Comment by hferentschik** [comment]
Cucumber integration test will be added as part of pull request #1228
